core/sql/regress/hive/EXPECTED020

```diff
@@ -132,24 +147,25 @@ S_STORE_SK   (EXPR)
 --- 1 row(s) selected.
 >>
 >>-- explain of join between 2 ORC tables
->>explain options 'f' select x.s_suite_number, y.s_street_name
+>>prepare XX from select x.s_suite_number, y.s_street_name
 +> from hive.hive.store_orc x, hive.hive.store_orc y
 +>  where x.s_store_sk = y.s_store_sk;
 
-LC   RC   OP   OPERATOR              OPT       DESCRIPTION           CARD
----- ---- ---- --------------------  --------  --------------------  ---------
+--- SQL command prepared.
+>>execute explainIt;
 
-3    .    4    root                                                  8.17E+004
-2    1    3    hybrid_hash_join                                      8.17E+004
-.    .    2    hive_scan                       STORE_ORC             4.08E+003
-.    .    1    hive_scan                       STORE_ORC             4.08E+003
+S   OPERATOR          LC  RC  TAB_NAME  
+--  ----------------  --  --  ----------
 
---- SQL operation complete.
+04  ROOT              3   ?             
+03  HYBRID_HASH_JOIN  2   1             
+02  HIVE_SCAN         ?   ?   .STORE_ORC
```

Comment:
Did you want the dot in front of the table name? (possible off-by-one error in the explainIt query text?) Can fix it later if desired.